runelite-client/src/main/java/net/runelite/client/ui/overlay/Overlay.java

```diff
@@ -31,7 +31,5 @@
 {
 	private OverlayPosition position = OverlayPosition.TOP_LEFT;
 	private OverlayPriority priority = OverlayPriority.NONE;
-	private boolean drawOverLoginScreen = false;
-	private boolean drawOverBankScreen = false;
-	private boolean drawOverClickToPlayScreen = false;
+	private OverlayLayer layer = OverlayLayer.ALWAYS_ON_TOP;
```

Comment:
OverlayLayer.ABOVE_WIDGETS would probably be better default layer, as it is most common place for overlays.